--- xslt-config.in.orig
+++ xslt-config.in
@@ -89,7 +89,7 @@ while test $# -gt 0; do
     shift
 done
 
-the_libs="@XSLT_LIBDIR@ @XSLT_LIBS@"
+the_libs="-lxslt"
 if test "$includedir" != "/usr/include"; then
     the_flags="$the_flags -I$includedir `@XML_CONFIG@ --cflags`"
 else
--- xsltConf.sh.in.orig
+++ xsltConf.sh.in
@@ -2,6 +2,6 @@
 # Configuration file for using the xslt library
 #
 XSLT_LIBDIR="@XSLT_LIBDIR@"
-XSLT_LIBS="@XSLT_LIBS@"
+XSLT_LIBS="-lxslt"
 XSLT_INCLUDEDIR="@XSLT_INCLUDEDIR@"
 MODULE_VERSION="xslt-@VERSION@"
